lading/src/generator/file_gen/logrotate_fs/model.rs

```diff
@@ -281,6 +272,20 @@ pub(crate) enum Node {
     },
 }
 
+/// Profile for load in this filesystem.
+#[derive(Debug, Clone, Copy)]
+pub(crate) enum LoadProfile {
```

Comment:
```suggestion
pub(crate) enum LoadProgression {
```

My thought is that the name should reflect that this defines how the load changes over time. "Profile" doesn't imply "over time", whereas progression does.

I also like `LoadGrowth`, and `Constant` would be `None` (no growth, just constant)

Reply:
I chose profile because I thought it was sort of neutral in the sense that we could incorporate things that don't change over time or don't have a sensible growth curve, like something that's just random.

This also made me think the block configuration should be wrapped up in a common spot and not repeated in every generator.